src/modules/gifmaker/GifRecorderOverlay/MainPage.xaml.cs

```diff
@@ -59,12 +62,69 @@ private void StopButton_Click(object sender, RoutedEventArgs e)
         {
             RecordPauseButton.IsEnabled = false;
             StopButton.IsEnabled = false;
+            MainCanvas.Children.Remove(Rect);
+            Rect = null;
             if ((string)RecordPauseButton.Tag == "pause")
             {
                 RecordPauseButton.Icon = new SymbolIcon(Symbol.Target);
                 RecordPauseButton.Label = "Record";
                 RecordPauseButton.Tag = "record";
             }
         }
+
+        private void MainCanvas_PointerPressed(object sender, PointerRoutedEventArgs e)
+        {
+            try
+            {
+                MainCanvas.CapturePointer(e.Pointer);
+                var ptrPoint = e.GetCurrentPoint(MainCanvas);
+                var point1 = ptrPoint.Position;
+                if (Rect != null)
+                {
+                    MainCanvas.Children.Remove(Rect);
+                }
+                Rect = new ResizableRectangle(MainCanvas, point1);
+                MainCanvas.Children.Add(Rect);
+                e.Handled = true;
+            }
+            catch (Exception)
+            {
+            }
+        }
+
+        private void MainCanvas_PointerMoved(object sender, PointerRoutedEventArgs e)
+        {
+            try
+            {
+                var ptrPoint = e.GetCurrentPoint(MainCanvas);
+                if (ptrPoint.Properties.IsLeftButtonPressed)
+                {
+                    if (Rect != null)
+                    {
+                        var point2 = ptrPoint.Position;
+                        Rect.SetCoordinates(point2);
+                    }
+                }
+            }
+            catch (Exception)
+            {
+            }
+        }
+
+        private void MainCanvas_PointerReleased(object sender, PointerRoutedEventArgs e)
+        {
+            try
+            {
+                if (Rect.IsZeroSize())
+                {
+                    MainCanvas.Children.Remove(Rect);
+                    Rect = null;
+                }
+                e.Handled = true;
+            }
+            catch (Exception)
+            {
+            }
```

Comment:
At the very least for the exception blocks can you write them to  the console in the case that something horrible happens?